packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -586,6 +586,56 @@ export const defaultErrorMessages = {
     });
   },
 
+  'component-has-static-classname-at-root': (
+    testInfo: IsConformantOptions,
+    error: Error,
+    componentClassName: string,
+    classNames: string,
+  ) => {
+    const { displayName } = testInfo;
+    const { testErrorInfo, resolveInfo, failedError } = errorMessageColors;
+
+    return getErrorMessage({
+      displayName,
+      overview: `does not have default className (${testErrorInfo(componentClassName)}).`,
+      details: [`After render it has following classes:`, `    ${failedError(`className='${classNames}'`)}`],
+      suggestions: [
+        `Ensure that your component has default className and it is ${resolveInfo('merged')} with defaults.`,
+      ],
+      error,
+    });
+  },
+
+  'component-has-static-classname-exported': (
+    testInfo: IsConformantOptions,
+    error: Error,
+    componentClassName: string,
+    exportName: string,
+  ) => {
+    const { componentPath, displayName } = testInfo;
+    const { testErrorInfo, resolveInfo, testErrorPath } = errorMessageColors;
+
+    const rootPath = componentPath.replace(/[\\/]src[\\/].*/, '');
```

Comment:
If this is an absolute path, you'll need to ensure you're replacing the last instance of `src` (we've had issues in the past where people put all their git repos in an `src` folder)

Reply:
I copied this from `exported-top-level`, is there anything wrong with it?

Reply:
@ecraig12345 I will address a problem if it exists in a follow up PR 👍

Reply:
It looks like there's a fix for this in `defaultTests.tsx` but not `defaultErrorMessages.tsx`

Reply:
So actually the `exported-top-level` error message would be wrong as well, but that would only be noticed if someone using `src` for their repos was developing a new component. (I think Levi was one of the people using this naming convention.)